uk.co.compucorp.civicrm.hrcontactactionsmenu/js/contactactions.js

```diff
@@ -36,11 +20,25 @@
     });
   }
 
+  /**
+   * Initializes listeners
```

Comment:
Please describe which listeners are initialised. That will answer on a question "what does this function do"

Reply:
Updated!